src/javascript/app_2/pages/trading/components/form/buttons.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const MdcButton = ({
+    id,
+    className='',
+    text,
+    is_ripple,
+    handleClick,
+}) => {
+    const classes = `${is_ripple ? `mdc-button mdc-button-ripple ${className}` : `mdc-button ${className}`}`;
```

Comment:
How about:
```js
const classes = `mdc-button${is_ripple ? ' mdc-button-ripple' : ''} ${className}`;
```
Or even more readable using [classNames](https://www.npmjs.com/package/classnames):
```js
const classes = classNames('mdc-button', {'mdc-button-ripple': is_ripple}, className);
```

Reply:
Thanks for pointing this out.